indexd/index/blueprint.py

```diff
@@ -264,6 +266,57 @@ def post_index_record():
     return flask.jsonify(ret), 200
 
 
+@blueprint.route('/index/blank/', methods=['POST'])
+@authorize
+def post_index_blank_record():
+    '''
+    Create a blank new record with only uploader field is filled
+    '''
+
+    uploader = flask.request.json.get('uploader')
```

Comment:
`request.get_json()` encouraged over `.json`

Reply:
Fixed in 7caea53ee872202df117db75459f21526c7b2db1